packages/rum-core/test/common/utils.spec.js

```diff
@@ -347,4 +347,34 @@ describe('lib/utils', function() {
     )
     expect(utils.removeInvalidChars('invalid"')).toEqual('invalid_')
   })
+
+  it('get servertiming info from servertiming entries', () => {
+    const serverTimingStr = utils.getServerTimingInfo([
+      {
+        name: 'cache',
+        duration: '200',
+        description: 'Origin cache'
+      },
+      {
+        name: 'edge',
+        duration: '20',
+        description: 'Edge cache'
+      },
+      {
+        name: 'miss'
```

Comment:
Can the name be undefined as well? i.e an empty object.

Reply:
No it cannot be as far i read the proposal. by default it will be parsed as name.